package.json

```diff
@@ -40,6 +40,7 @@
     "winston": "~1.0.0"
   },
   "devDependencies": {
+    "electron-packager": "^5.0.1",
```

Comment:
Any reason? Would it not be better to match minor versions, instead of just patches?